modules/ROOT/pages/installation/linux/rpm.adoc

```diff
@@ -64,6 +64,12 @@ gpgcheck=1
 EOF
 ----
 
+[NOTE]
+====
+If you are upgrading from Neo4j 4.4 or earlier, you may need to clear the package manager cache before neo4j packages become available:
```

Comment:
If they a moving from 4.4 to 5.x., they need to migrate. Also, from an earlier version than 4.4, migration is not supported. I actually think that this note should be added to the U&M guide instead of here. 
```suggestion
If you are upgrading from Neo4j 4.4 or earlier, you may need to clear the package manager cache before Neo4j packages become available:
```